db/migrate/20240610165140_update_dcyf_categories_and_eligibilties.rb

```diff
@@ -0,0 +1,285 @@
+class UpdateDcyfCategoriesAndEligibilties < ActiveRecord::Migration[6.1]
+  def up
+    ActiveRecord::Base.transaction do
+      # Set this to true to enable assertions
+      @assertions_enabled = true
+
+      ## Categories
+
+      # Arts and Creative Expression
+      ['Creative Writing', 'Music', 'Performing Arts', 'Photography and Film', 'Spoken Word', 'Theater', 'Visual Arts'].each do |from|
+        migrate_resources_and_services_to_new_category from: from, to: 'Arts and Creative Expression'
+        delete_category from
+      end
+
+      # Digital Media Production
+      rename_category from: 'Digital Arts', to: 'Digital Media Production'
+
+      # Disability Support
+      rename_category from: 'Disability', to: 'Disability Support'
+
+      # Justice Involvement
+      create_category 'Justice Involvement'
+
+      # LGBTQ+ Support
+      rename_category from: 'LGBTQ', to: 'LGBTQ+ Support'
+
+      # After & Before School Care
+      create_category 'After & Before School Care'
+
+      ['After School Care', 'Afterschool Programs', 'Before School Care'].each do |from|
+        migrate_resources_and_services_to_new_category from: from, to: 'After & Before School Care'
+        delete_category from
+      end
+
+      # Playgroups
+      create_category 'Playgroups'
+
+      # Academic Support
+      create_category 'Academic Support'
+
+      # TODO: Need to get the answer to two issues:
+      # 1) Summer School is not an actual category name
+      # 2) The spreadsheet says to _not_ delete Summer School, unlike all the
```

Comment:
Got confirmation that:

1. "Summer School" should actually be "Summer Programs", and
2. "Summer Programs" indeed should _not_ be deleted, unlike all the other ones. I'll update this code and make sure that we specifically do not delete "Summer Programs"

Reply:
Fixed in https://github.com/ShelterTechSF/askdarcel-api/pull/744/commits/30e74a9ffe6cc6fd996be1e0ca675a52fcb40e6c